app.gummi.gummi.yml

```diff
@@ -0,0 +1,90 @@
+id: app.gummi.gummi
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gummi
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    directory: texlive
+    subdirectories: true
+    autodelete: true
+    version: '22.08'
+
+finish-args:
+  - --share=ipc
+  - --filesystem=xdg-documents
+
+modules:
+  - shared-modules/intltool/intltool-0.51.json
+  
+  - name: mkdir-texlive
+    buildsystem: simple
+    build-commands:
+      - install -d /app/texlive
+  
+  - name: gtksourceview3
+    config-opts:
+      - --disable-gtk-doc
+      - --enable-maintainer-mode
+      - --enable-gtk-doc=no
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
+        sha256: 691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .;
+          - autoreconf -vfi;
+    post-install:
+      - install -p -D -m 0644 "COPYING" -t "${FLATPAK_DEST}/share/licenses/gtksourceview/";
+
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+      - "-DCMAKE_INSTALL_INCLUDEDIR=/app/include"
+      - "-DBUILD_GTK_TESTS=OFF"
+      - "-DBUILD_CPP_TESTS=OFF"
+      - "-DENABLE_CPP=OFF"
+      - "-DENABLE_BOOST=OFF"
```

Comment:
so you are 100% sure you never use Poppler splash backend?

Reply:
Removed flags at https://github.com/flathub/flathub/pull/3789/commits/2e8e228c230e97653fdd8c7d4af7353821c27130